src/test/ui/async-await/async-fn-size-moved-locals.rs

```diff
@@ -8,6 +8,7 @@
 // See issue #59123 for a full explanation.
 
 // ignore-emscripten (sizes don't match)
+// ignore-fuchsia (sizes don't match exactly)
```

Comment:
This seems unrelated to the PR's stated purpose, I'd prefer to split it (at least to a separate commit, ideally PR). I'm not sure why emscripten and fuchsia are special here, that seems surprising to me. (Maybe this is a needs-unwind in disguise?)

Reply:
This was accidentally included, I'll split this into a separate PR.